target/cheri-common/cheri-translate-utils.h

```diff
@@ -63,6 +65,8 @@ _gen_cap_check(rmw)
 
 #ifdef TARGET_MIPS
 static inline void gen_load_gpr(TCGv t, int reg);
+#elif defined(TARGET_AARCH64)
+    TCGv_i64 cpu_reg(DisasContext *s, int reg);
```

Comment:
There are a few TARGET_AARCH64 additions in here that should be held back. (Also there's unwanted indentation here)